assets/css/src/generic/_buttons.scss

```diff
@@ -1,29 +1,38 @@
 .mailpoet-button {
   align-items: center;
-  background: $color-secondary;
-  border: 0;
-  border-radius: 4px;
+  background: $color-primary;
+  border: 1px solid $color-primary;
+  border-radius: 2px;
   box-sizing: border-box;
   color: #fff;
   cursor: pointer;
   display: inline-flex;
   font-family: inherit;
-  font-size: $font-size;
-  font-weight: bold;
+  font-size: $font-size-button;
+  font-weight: normal;
   justify-content: center;
-  line-height: 24px;
+  line-height: 20px;
   max-width: 100%;
-  min-height: 40px;
-  padding: 9px 20px 7px;
+  min-height: 36px;
+  padding: 8px 12px 6px;
   position: relative;
   text-align: center;
   text-decoration: none;
   vertical-align: top;
 
-  &:hover,
+  &:hover {
+    background: $color-primary-hover;
+    border-color: $color-primary-hover;
+    color: #fff;
+    text-decoration: none;
+  }
+
   &:focus {
-    background: $color-secondary-hover;
+    background: $color-primary-hover;
+    border-color: #fff;
+    box-shadow: 0 0 0 1.5px $color-primary-hover;
     color: #fff;
+    outline: none;
     text-decoration: none;
```

Comment:
Do we need to keep all those styles? After adding WordPress button classes (`button-secondary` etc.) most of these rules are overwritten by CSS from WP. Does it makes sense to keep them?

![Screen Shot on 2021-06-03 at 11-21-48](https://user-images.githubusercontent.com/1082140/120623208-d2a68e00-c45f-11eb-85a6-66fa2688a5ef.png)